cmd/verify.go

```diff
@@ -54,27 +57,27 @@ var (
 	openshiftVersionFlag string
 )
 
-func filterChecks(set []string, subset []string, setEnabled bool, subsetEnabled bool) ([]string, error) {
-	selected := make([]string, 0)
-	seen := map[string]bool{}
-	for _, v := range set {
-		seen[v] = setEnabled
+func filterChecks(set map[checks.CheckName]checks.Check, subset []string, setEnabled bool, subsetEnabled bool) (map[checks.CheckName]checks.Check, error) {
```

Comment:
`map[checks.CheckName]checks.Check` gets repeated a lot; it likely means a type might be useful.

Perhaps the already existing registry type could be used as input and output, so the profile registry is the product of a function of the default registry and the profile description.

Reply:
OK I used 
DefaultRegistry type for  map[checks.CheckId]checks.Check

and:

Added in profiles.go: type FilteredRegistry map[checks.CheckName]checks.Check
Add in verifierBuilder.go: type FilteredRegistry map[checks.CheckName]checks.Check

I know these are the same but needed to avoid circular dependencies. Could just put the typoe in registry.go but that seemed inappropriate.